sw/device/lib/runtime/otbn.c

```diff
@@ -106,21 +92,11 @@ otbn_result_t otbn_load_app(otbn_t *ctx, const otbn_app_t app) {
   return kOtbnOk;
 }
 
-otbn_result_t otbn_call_function(otbn_t *ctx, otbn_ptr_t func) {
+otbn_result_t otbn_start(otbn_t *ctx) {
```

Comment:
Since we added support for more operations to OTBN, "start" isn't unambiguous any more. How about "otbn_execute()", matching the name of the operation (here and in other places in software).